benchmarks/hotel-app/yamls/docker-compose/dc-rate-tracing.yaml

```diff
@@ -0,0 +1,83 @@
+# MIT License
+
+# Copyright (c) 2022 EASE lab
+
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+version: "3.9"
+services:
+
+  ## Function ###
+  rate:
+    image: vhiveease/hotel-app-rate:latest
+    container_name: rate
+    entrypoint:
+      - /app/server
+      - --url=0.0.0.0
+      - --port=8083
+      - --db_addr=database:27017
+      - --zipkin=http://zipkin:9411/api/v2/spans
+    # ports:
+    # - published: 8083
+    #   target: 8083
+    environment:
+      - ENABLE_TRACING=true
+    depends_on:
+      - zipkin
+      - database
+      - memcached
+
+
+  ## Data base ##
+  database:
+    image: mongo:4.2
```

Comment:
please double check to use check vhiveease/ images for mongodb and memcached, etc